bedrock/firefox/views.py

```diff
@@ -226,10 +227,19 @@ def all_downloads(request, channel):
 
 
 def firefox_os_index(request):
+
+    locale = l10n_utils.get_locale(request)
+    lang_file = 'firefox/os/index-new'
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
     if waffle.switch_is_active('firefox-os-index-2015'):
```

Comment:
As my example showed earlier, you don't need two if statements for this. You just need a single:

```python
if waffle.switch_is_active('firefox-os-index-2015') and lang_file_is_active(lang_file, locale):
    return l10n_utils.render(request, new_home)
else:
    return l10n_utils.render(request, old_home)
```